### PR TITLE
doctrine result cache

### PR DESCRIPTION
**Pull request by sfarkas1988**
As i told you @Pierstoval, here the PR. I hope you like it. :)

### CODE REVIEW

**Comment by Pierstoval** [comment]
Wow, this PR is absolutely awesome!

Could you fix the issue in tests for Symfony 2.3 ? :)

**Comment by sfarkas1988** [comment]
@Pierstoval Any idea how to fix the current error to make it work for 3.0 and 2.3 ?
My last idea is not ideal but i would pass the config for each repository method instead using the repository as a service.

**Comment by sfarkas1988** [comment]
hell yeah, all checks passed, didn't expect so many problems, but great that you use these tools. :+1:

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ composer.lock
 vendor/
 build/
 phpunit.xml
+.idea
```
Comment (Pierstoval) [inline .gitignore]: This should be in your global `.gitignore` file :)

File: README.md [hunk 2]
```diff
@@ -278,6 +278,30 @@ Basically, it will look like this:
...
+        orm:
+            result_cache_driver: apc            
+```
+
+You can read more about DoctrineCache <a href="http://symfony.com/doc/current/reference/configuration/doctrine.html#caching-drivers">here</a>.
+
+
+
+
+
```
Comment (Pierstoval) [inline README.md]: Too many spaces here, could you reduce to one single space? 
There's a title the line below so it's already separated ;)

**Comment by Pierstoval** [comment]
Apart the few comments I made, it seems good to me, once you make these little fixes I'm gonna merge this, add a bit of CS fixer to it and release a new version :)
Also, could you rebase and squash all your commits before I merge?

**Comment by sfarkas1988** [comment]
done :+1:

**Comment by Pierstoval** [comment]
Still my [comment](#discussion_r47495533) is not fixed :) 

As an information, platform-specific files should be ignored by your global git configuration.

Check [this gist](https://gist.github.com/Pierstoval/a27097000e1e71242e90), the `core.excludesfile` statement adds a global `.gitignore` file in which you should add things like `.idea` for JetBrains IDEs, but also `.DS_STORE` on Mac OSX, or `thumbs.db` for Windows, or even `*.php~` (for backed-up entities from the `generate:entities` command, for example).

**Comment by sfarkas1988** [comment]
oh yes, sure. but now for sure :D

**Pull request merged**